core/utils.py

```diff
@@ -1365,3 +1365,53 @@ def url_open(source_url: str) -> urllib.request._UrlopenRet:
     # check is refactored.
     context = ssl.create_default_context(cafile=certifi.where())  # pylint: disable=arg-name-for-non-keyword-arg
     return urllib.request.urlopen(source_url, context=context)
+
+
+def escape_html(unescaped_html_data: str) -> str:
+    """This functions escapes an unescaped HTML string.
+
+    Args:
+        unescaped_html_data: str. Unescaped HTML string to be escaped.
+
+    Returns:
+        str. Escaped HTML string.
+    """
+    # Replace list to escape html strings.
+    replace_list_for_escaping = [
+        ('&', '&amp;'),
+        ('"', '&quot;'),
+        ('\'', '&#39;'),
+        ('<', '&lt;'),
+        ('>', '&gt;')
+    ]
+    escaped_html_data = unescaped_html_data
+    for replace_tuple in replace_list_for_escaping:
+        escaped_html_data = escaped_html_data.replace(
+            replace_tuple[0], replace_tuple[1])
+
+    return escaped_html_data
```

Comment:
Should unit tests be added for these? Or moved from some other file?

Reply:
I was not able to find them but I have created new unit tests and added them.
Thanks